src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx

```diff
@@ -34,14 +35,16 @@ export const PullListByRepo = ({
   isRepoOpen
 }: PullListByRepoProps) => {
   const { translate } = useDictionary()
-  const { toggleRepoCheckStatus, togglePullCheckStatus, toggleRepoOpenStatus } =
-    useUpdateRepoOrPull()
+  const {
+    /* toggleRepoCheckStatus, */ togglePullCheckStatus,
+    toggleRepoOpenStatus
+  } = useUpdateRepoOrPull()
   // TODO: handle error case
-  const { isPending, isLoading } = useFetchPulls(owner, repo, isRepoOpen)
-  const hasChild = !!pulls && pulls.length > 0
-  const isRepoChecked = !!pulls
-    ? pulls.length > 0 && pulls.every((pull) => pull.checked)
-    : false
+  const { /* isPending, */ isLoading } = useFetchPulls(owner, repo, isRepoOpen)
```

Comment:
Can you remove all of these comments? It's very difficult to review. If you put it temporary, backup somehow on your local machine using stash or in a different branch.

Reply:
Ah sorry, I forgot it